packages/flutter_tools/lib/src/commands/build_bundle.dart

```diff
@@ -102,6 +103,7 @@ class BuildBundleCommand extends BuildSubCommand {
           throwToolExit('macOS is not a supported target platform.');
         }
       case TargetPlatform.windows_x64:
+      case TargetPlatform.windows_arm64:
```

Comment:
Let's make `flutter build bundle --target-platform windows-arm64` error if used on the beta/stable channels.

Reply:
Added this check.